src/sinks/datadog_metrics.rs

```diff
@@ -225,6 +237,41 @@ fn encode_namespace(namespace: &str, name: &str) -> String {
     }
 }
 
+fn stats(values: &Vec<f64>, counts: &Vec<u32>) -> Option<DatadogStats> {
+    let mut samples = Vec::new();
+    for (v, c) in values.iter().zip(counts.iter()) {
+        for _ in 0..*c {
+            samples.push(*v);
+        }
+    }
+
+    if samples.is_empty() {
+        return None;
+    }
+
+    samples.sort_by(|a, b| a.partial_cmp(b).unwrap());
```

Comment:
It's probably super unlikely that this ever causes a problem, but would it be a little safer to do `unwrap_or(Ordering::Equal)` or something like that?